src/browser/AxePage.ts

```diff
@@ -26,13 +27,24 @@ function runAxe(options: RunOptions): Promise<AxeResults> {
   return window.axeQueue.add(() => window.axe.run(document, options));
 }
 
-function getOptions(options: RunOptions, disabledRules: string[] = []) {
+export function getRunOptions(
+  options: RunOptions,
+  disabledRules: string[] = [],
+): RunOptions {
   const newRules: RuleObject = options.rules || {};
 
   for (const rule of disabledRules) {
     newRules[rule] = { enabled: false };
   }
 
+  /**
+   * TODO-ah: if `options.rules` AND disabledRules exist, we should:
+   * - merge the two together (preserves API and call signatures)
+   * - overwrite any rules from `runOptions` with those based on defaults and `disabledRules`
+   * Also:
+   * - throw a warning in case of deprecation approach?
+   * - throw an error in case of deprecation approach?
+   *  */
```

Comment:
@ahuth Any thoughts on handling here? I implemented the overwrite option but wasn't sure if we want to do more here like mark using `disabledRules` as deprecated, etc.

I can remove this comment if we are good with the overwrite

Reply:
I'm definitely good with the overwrite, since it's simple and clear.